Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -176,24 +186,26 @@ extension XcodeModuleProtocol  {
         return buildSettings.map{ "\($0) = '\($1)';" }.joined(separator: " ")
     }
 
-    func getReleaseBuildSettings(_ options: XcodeprojOptions) -> String {
-        var buildSettings = getCommonBuildSettings(options)
+    func getReleaseBuildSettings(_ options: XcodeprojOptions, xcodeProjectPath: String) -> String {
+        var buildSettings = getCommonBuildSettings(options, xcodeProjectPath: xcodeProjectPath)
         if let headerSearchPaths = headerSearchPaths {
             buildSettings[headerSearchPaths.key] = headerSearchPaths.value
         }
         // FIXME: Need to honor actual quoting rules here.
         return buildSettings.map{ "\($0) = '\($1)';" }.joined(separator: " ")
     }
 
-    private func getCommonBuildSettings(_ options: XcodeprojOptions) ->[String: String] {
+    private func getCommonBuildSettings(_ options: XcodeprojOptions, xcodeProjectPath: String) -> [String: String] {
         var buildSettings = [String: String]()
+        let plistPath = Path.join(xcodeProjectPath, infoPlistFileName)
```

Comment:
Xcode will resolve the plist path relative to the project directory it is referenced from. We should probably use that fact to only generate a relative path here, so that projects _can_ be relocated, even though the intent is that they be regenerated.